### PR TITLE
fix(DB/creature_template): Improve Naxxramas

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Raid.

How to test:
Apply SQL
Restart Server and delete your Cache.
Make sure to add basic tanking gear towards Raid ilevel.

Before you start checking damage:
.mod hp 500000
.learn 1908 (Uber heal over time)

If there is any issue with damage and if you get instantly killed, lemme know.

Note: You wont be able to survive some mechanics where second tank is required - Use one more account on those fights.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1609681055620795500.sql [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+/* THE CONSTRUCT QUARTER */
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 16028;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29324;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15931;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29373;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15932;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29417;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15928;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29448;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609681055620795500.sql:116]: You dont need so many queries if you group with entry IN (X,Y) with those that you update the same things

File: data/sql/updates/pending_db_world/rev_1609681055620795500.sql [hunk 2]
```diff
@@ -0,0 +1,122 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609681055620795500');
+/*
+ * Raid: The Naxxramas
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1609681055620795500.sql:5]: better remove this as they will be deleted anyway, you can add them in the PR description which instead stays there forever

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1609681055620795500.sql]: Sorted xP

It's my fix template.

File: data/sql/updates/pending_db_world/rev_1609681055620795500.sql (hunk 1 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1609681055620795500.sql:110]: Some of them might get updated as always.
Not gonna re-create again if use same for multiple.

It's always better to keep those separate cause they are not some random mobs and they effect game a lot.

**Review by FrancescoBorzi — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1609681055620795500.sql [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15954;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29615;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15936;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29701;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 16011;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29718;
+
+/* THE MILITARY QUARTER */
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 16061;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29940;
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1609681055620795500.sql:90]: Nice joke, now where is the damage?

let's take example of this clip...

Instructor razuvious (10-mode) **HITS** a tank of 3.7k GS who has active "Shield wall" which reduces damage taken by 60%, with a basic melee attack... and does **10,256 physical damage.**

where is the total?

let's calculate that...

total MINUS 60% of Shield Wall = damage to character

damage to character MINUS 60% physical damage reduce by flat armor = damage taken

damage taken = 10,256

image

image

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1609681055620795500.sql:90]: I'm working on it, told ya to not worry about it.

We doing full checks, cause some of them will need even lower due mechanics.

Comment (Si1ker) [reply data/sql/updates/pending_db_world/rev_1609681055620795500.sql:90]: "We doing full checks, cause some of them will need even lower due mechanics."

Hope they will come with full and detailed explanation on those cases ..   : )

**Comment by pklloveyou** [comment]
mark

**Comment by Knindzagxg** [comment]
Its still work in progress.
Due few more other things, this one should be completed in 2 weeks.

**Comment by Knindzagxg** [comment]
@Si1ker 

Need testing and checking for Instructor.
Damage on tank will be good now, just dunno if "Guardians" gonna be able to tank with Bone Armor.

**Comment by Si1ker** [comment]
Ok @Knindzagxg , will check/test

**Comment by claudiodfc** [comment]
Is this PR still viable after the creature level stats PR?

**Comment by FrancescoBorzi** [comment]
@Knindzagxg the build is failing

**Comment by Knindzagxg** [comment]
This wil be closed, im just gonna test new DB changes during weekend, since we have new system.

**Pull request closed without merging**